homework_04/05_viktoriya_vasileva/homework_04_05_viktoriya_vasileva.yaml

```diff
@@ -0,0 +1,34 @@
+name: Pylint
```

Comment:
The convention here should be: Homework 04 - xx Firstname Lastname

Furthermore, this file should be in the folder .github/workflows.